SAMtags.tex

```diff
@@ -453,6 +455,112 @@ \subsubsection{Color space}
 Color read quality on the original strand of the read. Same encoding as {\sf QUAL}; same length as {\tt CS}.
 \end{description}
 
+\subsection{Base modifications}
+
+Base modifications, including base methylation, are represented as a
+series of edits from the primary unmodified sequence stored in the
+main SAM {\sf SEQ} field.  If the modifed base has no natural
```

Comment:
I find this still to be unclear in the presence of alignment...I think that we should store a representation of edits to the originally sequenced (i.e. unaligned, and un-reverse complemented) bases.

Reply:
ah...I see that the details explains...